### PR TITLE
Reduce the default size

### PR DESCRIPTION
**Pull request by taoyuanyuan**
### Issues associated with this PR

Your PR should present related issues you want to solve.

### Sign the CLA
Make sure you have signed the [CLA](https://www.clahub.com/agreements/alipay/sofa-mosn)

### Solutions
You should show your solutions about the issues in your PR, including the overall solutions, 
details and the changes. At this time, Chinese is allowed to describe these.

### UT result
Unit Test is needed if the code is changed, your unit test should cover boundary cases, corner cases, and some exceptional cases.
And you need to show the ut result.

### Benchmark
If your code involves the processing of every request, you should give the Benchmark Result

### Code Style
+ Make sure `Goimports` has run
+ Show `Golint` result

### CODE REVIEW

File: pkg/log/logger.go [hunk 1]
```diff
@@ -92,7 +92,7 @@ func GetOrCreateLogger(output string, roller *Roller) (*Logger, error) {
 	lg := &Logger{
 		output:          output,
 		roller:          roller,
-		writeBufferChan: make(chan types.IoBuffer, 1000),
+		writeBufferChan: make(chan types.IoBuffer, 500),
```
Comment (nejisama) [inline pkg/log/logger.go]: 日志的buffer改小了，主站并发多的话，会不会有影响

Comment (taoyuanyuan) [reply pkg/log/logger.go]: 500已经很大了，量大了就丢弃
